lib/taskbook.js

```diff
@@ -307,13 +307,16 @@ class Taskbook {
   checkTasks(ids) {
     ids = this._validateIDs(ids);
     const {_data} = this;
+    const [checked, unChecked] = [[], []];
```

Comment:
It is completely Ok to avoid the camel-case here and go for `unchecked` as a variable name:

```js
const [checked, unchecked] = [[], []];
```